cmd/frontend/graphqlbackend/site_alerts.go

```diff
@@ -127,3 +138,81 @@ func init() {
 		return alerts
 	})
 }
+
+func outOfDateAlert(isAdmin bool) Alert {
+
+	var alert Alert
+	globalUpdateStatus := updatecheck.Last()
+	if globalUpdateStatus == nil || updatecheck.IsPending() {
+		return alert
+	}
+	months := globalUpdateStatus.MonthsOutOfDate
+	if months <= 0 {
+		return alert
+	}
+	// online instances will still be prompt prompt side admins to upgrade via site_update_check
+	if months < 3 && !globalUpdateStatus.Offline {
+		return alert
+	}
+
+	if isAdmin {
+		alert.MessageValue = fmt.Sprintf("Sourcegraph is %d months out of date", months)
+		key := fmt.Sprintf("%d", months)
```

Comment:
```suggestion
		key := fmt.Sprintf("months-out-of-date-%d", months)
```

This key actually needs to be globally unique (it's just a browser local-storage key)